[2025-04-21T02:06:29.536-0300] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-21T02:06:29.558-0300] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: DW.Duck scheduled__2025-04-20T00:00:00+00:00 [queued]>
[2025-04-21T02:06:29.567-0300] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: DW.Duck scheduled__2025-04-20T00:00:00+00:00 [queued]>
[2025-04-21T02:06:29.568-0300] {taskinstance.py:2866} INFO - Starting attempt 1 of 4
[2025-04-21T02:06:29.594-0300] {taskinstance.py:2889} INFO - Executing <Task(BashOperator): Duck> on 2025-04-20 00:00:00+00:00
[2025-04-21T02:06:29.606-0300] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'DW', 'Duck', 'scheduled__2025-04-20T00:00:00+00:00', '--job-id', '28', '--raw', '--subdir', 'DAGS_FOLDER/DW.py', '--cfg-path', '/tmp/tmpx33fgalp']
[2025-04-21T02:06:29.607-0300] {standard_task_runner.py:105} INFO - Job 28: Subtask Duck
[2025-04-21T02:06:29.614-0300] {logging_mixin.py:190} WARNING - /home/lenovo/miniconda3/envs/worker/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=15845) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-04-21T02:06:29.614-0300] {standard_task_runner.py:72} INFO - Started process 15950 to run task
[2025-04-21T02:06:29.668-0300] {task_command.py:467} INFO - Running <TaskInstance: DW.Duck scheduled__2025-04-20T00:00:00+00:00 [running]> on host DESKTOP-1GIT60U.
[2025-04-21T02:06:29.773-0300] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='DW' AIRFLOW_CTX_TASK_ID='Duck' AIRFLOW_CTX_EXECUTION_DATE='2025-04-20T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-04-20T00:00:00+00:00'
[2025-04-21T02:06:29.775-0300] {taskinstance.py:731} INFO - ::endgroup::
[2025-04-21T02:06:29.794-0300] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-04-21T02:06:29.796-0300] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'python3 /home/lenovo/airflow/jobs/jobs_functions/dw_motherduck.py']
[2025-04-21T02:06:29.811-0300] {subprocess.py:99} INFO - Output:
[2025-04-21T02:06:30.685-0300] {subprocess.py:106} INFO - Traceback (most recent call last):
[2025-04-21T02:06:30.685-0300] {subprocess.py:106} INFO -   File "/home/lenovo/airflow/jobs/jobs_functions/dw_motherduck.py", line 23, in <module>
[2025-04-21T02:06:30.686-0300] {subprocess.py:106} INFO -     from jobs.s3.s3_functions import *
[2025-04-21T02:06:30.686-0300] {subprocess.py:106} INFO - ModuleNotFoundError: No module named 'jobs'
[2025-04-21T02:06:30.779-0300] {subprocess.py:110} INFO - Command exited with return code 1
[2025-04-21T02:06:30.789-0300] {taskinstance.py:3311} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/lenovo/miniconda3/envs/worker/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lenovo/miniconda3/envs/worker/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lenovo/miniconda3/envs/worker/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lenovo/miniconda3/envs/worker/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 422, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lenovo/miniconda3/envs/worker/lib/python3.12/site-packages/airflow/operators/bash.py", line 276, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 1.
[2025-04-21T02:06:30.797-0300] {taskinstance.py:1225} INFO - Marking task as UP_FOR_RETRY. dag_id=DW, task_id=Duck, run_id=scheduled__2025-04-20T00:00:00+00:00, execution_date=20250420T000000, start_date=20250421T050629, end_date=20250421T050630
[2025-04-21T02:06:30.839-0300] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-04-21T02:06:30.840-0300] {standard_task_runner.py:124} ERROR - Failed to execute job 28 for task Duck (Bash command failed. The command returned a non-zero exit code 1.; 15950)
Traceback (most recent call last):
  File "/home/lenovo/miniconda3/envs/worker/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py", line 117, in _start_by_fork
    ret = args.func(args, dag=self.dag)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lenovo/miniconda3/envs/worker/lib/python3.12/site-packages/airflow/cli/cli_config.py", line 49, in command
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/lenovo/miniconda3/envs/worker/lib/python3.12/site-packages/airflow/utils/cli.py", line 116, in wrapper
    return f(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^
  File "/home/lenovo/miniconda3/envs/worker/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 483, in task_run
    task_return_code = _run_task_by_selected_method(args, _dag, ti)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lenovo/miniconda3/envs/worker/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 256, in _run_task_by_selected_method
    return _run_raw_task(args, ti)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lenovo/miniconda3/envs/worker/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 341, in _run_raw_task
    return ti._run_raw_task(
           ^^^^^^^^^^^^^^^^^
  File "/home/lenovo/miniconda3/envs/worker/lib/python3.12/site-packages/airflow/utils/session.py", line 97, in wrapper
    return func(*args, session=session, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lenovo/miniconda3/envs/worker/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3005, in _run_raw_task
    return _run_raw_task(
           ^^^^^^^^^^^^^^
  File "/home/lenovo/miniconda3/envs/worker/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 273, in _run_raw_task
    TaskInstance._execute_task_with_callbacks(
  File "/home/lenovo/miniconda3/envs/worker/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3159, in _execute_task_with_callbacks
    result = self._execute_task(context, task_orig)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lenovo/miniconda3/envs/worker/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3183, in _execute_task
    return _execute_task(self, context, task_orig)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lenovo/miniconda3/envs/worker/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lenovo/miniconda3/envs/worker/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lenovo/miniconda3/envs/worker/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lenovo/miniconda3/envs/worker/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 422, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lenovo/miniconda3/envs/worker/lib/python3.12/site-packages/airflow/operators/bash.py", line 276, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 1.
[2025-04-21T02:06:30.874-0300] {local_task_job_runner.py:266} INFO - Task exited with return code 1
[2025-04-21T02:06:30.892-0300] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-04-21T02:06:30.893-0300] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-04-21T02:15:19.115-0300] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-21T02:15:19.134-0300] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: DW.Duck scheduled__2025-04-20T00:00:00+00:00 [queued]>
[2025-04-21T02:15:19.145-0300] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: DW.Duck scheduled__2025-04-20T00:00:00+00:00 [queued]>
[2025-04-21T02:15:19.146-0300] {taskinstance.py:2866} INFO - Starting attempt 1 of 4
[2025-04-21T02:15:19.171-0300] {taskinstance.py:2889} INFO - Executing <Task(BashOperator): Duck> on 2025-04-20 00:00:00+00:00
[2025-04-21T02:15:19.184-0300] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'DW', 'Duck', 'scheduled__2025-04-20T00:00:00+00:00', '--job-id', '39', '--raw', '--subdir', 'DAGS_FOLDER/DW.py', '--cfg-path', '/tmp/tmpysimmic3']
[2025-04-21T02:15:19.186-0300] {standard_task_runner.py:105} INFO - Job 39: Subtask Duck
[2025-04-21T02:15:19.191-0300] {logging_mixin.py:190} WARNING - /home/lenovo/miniconda3/envs/worker/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=19803) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-04-21T02:15:19.192-0300] {standard_task_runner.py:72} INFO - Started process 19893 to run task
[2025-04-21T02:15:19.240-0300] {task_command.py:467} INFO - Running <TaskInstance: DW.Duck scheduled__2025-04-20T00:00:00+00:00 [running]> on host DESKTOP-1GIT60U.
[2025-04-21T02:15:19.336-0300] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='DW' AIRFLOW_CTX_TASK_ID='Duck' AIRFLOW_CTX_EXECUTION_DATE='2025-04-20T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-04-20T00:00:00+00:00'
[2025-04-21T02:15:19.337-0300] {taskinstance.py:731} INFO - ::endgroup::
[2025-04-21T02:15:19.355-0300] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-04-21T02:15:19.356-0300] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'PYTHONPATH=/home/lenovo/airflow:/home/lenovo/airflow/jobs python3 /home/lenovo/airflow/jobs/jobs_functions/dw_motherduck.py']
[2025-04-21T02:15:19.371-0300] {subprocess.py:99} INFO - Output:
[2025-04-21T02:15:19.667-0300] {subprocess.py:106} INFO - Traceback (most recent call last):
[2025-04-21T02:15:19.667-0300] {subprocess.py:106} INFO -   File "/home/lenovo/airflow/jobs/jobs_functions/dw_motherduck.py", line 15, in <module>
[2025-04-21T02:15:19.668-0300] {subprocess.py:106} INFO -     from pyspark.sql import SparkSession
[2025-04-21T02:15:19.668-0300] {subprocess.py:106} INFO - ModuleNotFoundError: No module named 'pyspark'
[2025-04-21T02:15:19.702-0300] {subprocess.py:110} INFO - Command exited with return code 1
[2025-04-21T02:15:19.712-0300] {taskinstance.py:3311} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/lenovo/miniconda3/envs/worker/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lenovo/miniconda3/envs/worker/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lenovo/miniconda3/envs/worker/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lenovo/miniconda3/envs/worker/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 422, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lenovo/miniconda3/envs/worker/lib/python3.12/site-packages/airflow/operators/bash.py", line 276, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 1.
[2025-04-21T02:15:19.721-0300] {taskinstance.py:1225} INFO - Marking task as UP_FOR_RETRY. dag_id=DW, task_id=Duck, run_id=scheduled__2025-04-20T00:00:00+00:00, execution_date=20250420T000000, start_date=20250421T051519, end_date=20250421T051519
[2025-04-21T02:15:19.765-0300] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-04-21T02:15:19.766-0300] {standard_task_runner.py:124} ERROR - Failed to execute job 39 for task Duck (Bash command failed. The command returned a non-zero exit code 1.; 19893)
Traceback (most recent call last):
  File "/home/lenovo/miniconda3/envs/worker/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py", line 117, in _start_by_fork
    ret = args.func(args, dag=self.dag)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lenovo/miniconda3/envs/worker/lib/python3.12/site-packages/airflow/cli/cli_config.py", line 49, in command
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/lenovo/miniconda3/envs/worker/lib/python3.12/site-packages/airflow/utils/cli.py", line 116, in wrapper
    return f(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^
  File "/home/lenovo/miniconda3/envs/worker/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 483, in task_run
    task_return_code = _run_task_by_selected_method(args, _dag, ti)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lenovo/miniconda3/envs/worker/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 256, in _run_task_by_selected_method
    return _run_raw_task(args, ti)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lenovo/miniconda3/envs/worker/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 341, in _run_raw_task
    return ti._run_raw_task(
           ^^^^^^^^^^^^^^^^^
  File "/home/lenovo/miniconda3/envs/worker/lib/python3.12/site-packages/airflow/utils/session.py", line 97, in wrapper
    return func(*args, session=session, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lenovo/miniconda3/envs/worker/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3005, in _run_raw_task
    return _run_raw_task(
           ^^^^^^^^^^^^^^
  File "/home/lenovo/miniconda3/envs/worker/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 273, in _run_raw_task
    TaskInstance._execute_task_with_callbacks(
  File "/home/lenovo/miniconda3/envs/worker/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3159, in _execute_task_with_callbacks
    result = self._execute_task(context, task_orig)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lenovo/miniconda3/envs/worker/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3183, in _execute_task
    return _execute_task(self, context, task_orig)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lenovo/miniconda3/envs/worker/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lenovo/miniconda3/envs/worker/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lenovo/miniconda3/envs/worker/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lenovo/miniconda3/envs/worker/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 422, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lenovo/miniconda3/envs/worker/lib/python3.12/site-packages/airflow/operators/bash.py", line 276, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 1.
[2025-04-21T02:15:19.809-0300] {local_task_job_runner.py:266} INFO - Task exited with return code 1
[2025-04-21T02:15:19.829-0300] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-04-21T02:15:19.830-0300] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-04-21T02:28:51.453-0300] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-21T02:28:51.471-0300] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: DW.Duck scheduled__2025-04-20T00:00:00+00:00 [queued]>
[2025-04-21T02:28:51.482-0300] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: DW.Duck scheduled__2025-04-20T00:00:00+00:00 [queued]>
[2025-04-21T02:28:51.483-0300] {taskinstance.py:2866} INFO - Starting attempt 1 of 4
[2025-04-21T02:28:51.513-0300] {taskinstance.py:2889} INFO - Executing <Task(BashOperator): Duck> on 2025-04-20 00:00:00+00:00
[2025-04-21T02:28:51.525-0300] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'DW', 'Duck', 'scheduled__2025-04-20T00:00:00+00:00', '--job-id', '44', '--raw', '--subdir', 'DAGS_FOLDER/DW.py', '--cfg-path', '/tmp/tmpwtwdgpom']
[2025-04-21T02:28:51.526-0300] {standard_task_runner.py:105} INFO - Job 44: Subtask Duck
[2025-04-21T02:28:51.531-0300] {logging_mixin.py:190} WARNING - /home/lenovo/miniconda3/envs/worker/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=23384) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-04-21T02:28:51.532-0300] {standard_task_runner.py:72} INFO - Started process 23486 to run task
[2025-04-21T02:28:51.583-0300] {task_command.py:467} INFO - Running <TaskInstance: DW.Duck scheduled__2025-04-20T00:00:00+00:00 [running]> on host DESKTOP-1GIT60U.
[2025-04-21T02:28:51.683-0300] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='DW' AIRFLOW_CTX_TASK_ID='Duck' AIRFLOW_CTX_EXECUTION_DATE='2025-04-20T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-04-20T00:00:00+00:00'
[2025-04-21T02:28:51.685-0300] {taskinstance.py:731} INFO - ::endgroup::
[2025-04-21T02:28:51.702-0300] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-04-21T02:28:51.704-0300] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'python -m jobs.jobs_functions.dw_motherduck']
[2025-04-21T02:28:51.724-0300] {subprocess.py:99} INFO - Output:
[2025-04-21T02:28:51.745-0300] {subprocess.py:106} INFO - /home/lenovo/miniconda3/envs/worker/bin/python: Error while finding module specification for 'jobs.jobs_functions.dw_motherduck' (ModuleNotFoundError: No module named 'jobs')
[2025-04-21T02:28:51.749-0300] {subprocess.py:110} INFO - Command exited with return code 1
[2025-04-21T02:28:51.759-0300] {taskinstance.py:3311} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/lenovo/miniconda3/envs/worker/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lenovo/miniconda3/envs/worker/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lenovo/miniconda3/envs/worker/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lenovo/miniconda3/envs/worker/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 422, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lenovo/miniconda3/envs/worker/lib/python3.12/site-packages/airflow/operators/bash.py", line 276, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 1.
[2025-04-21T02:28:51.767-0300] {taskinstance.py:1225} INFO - Marking task as UP_FOR_RETRY. dag_id=DW, task_id=Duck, run_id=scheduled__2025-04-20T00:00:00+00:00, execution_date=20250420T000000, start_date=20250421T052851, end_date=20250421T052851
[2025-04-21T02:28:51.795-0300] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-04-21T02:28:51.796-0300] {standard_task_runner.py:124} ERROR - Failed to execute job 44 for task Duck (Bash command failed. The command returned a non-zero exit code 1.; 23486)
Traceback (most recent call last):
  File "/home/lenovo/miniconda3/envs/worker/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py", line 117, in _start_by_fork
    ret = args.func(args, dag=self.dag)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lenovo/miniconda3/envs/worker/lib/python3.12/site-packages/airflow/cli/cli_config.py", line 49, in command
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/lenovo/miniconda3/envs/worker/lib/python3.12/site-packages/airflow/utils/cli.py", line 116, in wrapper
    return f(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^
  File "/home/lenovo/miniconda3/envs/worker/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 483, in task_run
    task_return_code = _run_task_by_selected_method(args, _dag, ti)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lenovo/miniconda3/envs/worker/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 256, in _run_task_by_selected_method
    return _run_raw_task(args, ti)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lenovo/miniconda3/envs/worker/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 341, in _run_raw_task
    return ti._run_raw_task(
           ^^^^^^^^^^^^^^^^^
  File "/home/lenovo/miniconda3/envs/worker/lib/python3.12/site-packages/airflow/utils/session.py", line 97, in wrapper
    return func(*args, session=session, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lenovo/miniconda3/envs/worker/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3005, in _run_raw_task
    return _run_raw_task(
           ^^^^^^^^^^^^^^
  File "/home/lenovo/miniconda3/envs/worker/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 273, in _run_raw_task
    TaskInstance._execute_task_with_callbacks(
  File "/home/lenovo/miniconda3/envs/worker/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3159, in _execute_task_with_callbacks
    result = self._execute_task(context, task_orig)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lenovo/miniconda3/envs/worker/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3183, in _execute_task
    return _execute_task(self, context, task_orig)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lenovo/miniconda3/envs/worker/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lenovo/miniconda3/envs/worker/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lenovo/miniconda3/envs/worker/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lenovo/miniconda3/envs/worker/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 422, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lenovo/miniconda3/envs/worker/lib/python3.12/site-packages/airflow/operators/bash.py", line 276, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 1.
[2025-04-21T02:28:51.829-0300] {local_task_job_runner.py:266} INFO - Task exited with return code 1
[2025-04-21T02:28:51.853-0300] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-04-21T02:28:51.854-0300] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-04-21T02:37:37.308-0300] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-21T02:37:37.327-0300] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: DW.Duck scheduled__2025-04-20T00:00:00+00:00 [queued]>
[2025-04-21T02:37:37.336-0300] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: DW.Duck scheduled__2025-04-20T00:00:00+00:00 [queued]>
[2025-04-21T02:37:37.336-0300] {taskinstance.py:2866} INFO - Starting attempt 1 of 4
[2025-04-21T02:37:37.365-0300] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): Duck> on 2025-04-20 00:00:00+00:00
[2025-04-21T02:37:37.374-0300] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'DW', 'Duck', 'scheduled__2025-04-20T00:00:00+00:00', '--job-id', '49', '--raw', '--subdir', 'DAGS_FOLDER/DW.py', '--cfg-path', '/tmp/tmpnx1f0113']
[2025-04-21T02:37:37.376-0300] {standard_task_runner.py:105} INFO - Job 49: Subtask Duck
[2025-04-21T02:37:37.381-0300] {logging_mixin.py:190} WARNING - /home/lenovo/miniconda3/envs/worker/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=26593) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-04-21T02:37:37.381-0300] {standard_task_runner.py:72} INFO - Started process 26690 to run task
[2025-04-21T02:37:37.424-0300] {task_command.py:467} INFO - Running <TaskInstance: DW.Duck scheduled__2025-04-20T00:00:00+00:00 [running]> on host DESKTOP-1GIT60U.
[2025-04-21T02:37:37.525-0300] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='DW' AIRFLOW_CTX_TASK_ID='Duck' AIRFLOW_CTX_EXECUTION_DATE='2025-04-20T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-04-20T00:00:00+00:00'
[2025-04-21T02:37:37.527-0300] {taskinstance.py:731} INFO - ::endgroup::
[2025-04-21T03:01:38.926-0300] {local_task_job_runner.py:346} WARNING - State of this instance has been externally set to None. Terminating instance.
[2025-04-21T03:01:38.930-0300] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-04-21T03:01:38.934-0300] {process_utils.py:132} INFO - Sending 15 to group 26690. PIDs of all processes in the group: [26690]
[2025-04-21T03:01:38.935-0300] {process_utils.py:87} INFO - Sending the signal 15 to group 26690
[2025-04-21T03:02:42.710-0300] {process_utils.py:150} WARNING - process psutil.Process(pid=26690, name='airflow task runner: DW Duck scheduled__2025-04-20T00:00:00+00:00 49', status='sleeping', started='02:37:36') did not respond to SIGTERM. Trying SIGKILL
[2025-04-21T03:02:42.713-0300] {process_utils.py:87} INFO - Sending the signal 9 to group 26690
[2025-04-21T03:02:42.727-0300] {process_utils.py:80} INFO - Process psutil.Process(pid=26690, name='airflow task runner: DW Duck scheduled__2025-04-20T00:00:00+00:00 49', status='terminated', exitcode=<Negsignal.SIGKILL: -9>, started='02:37:36') (26690) terminated with exit code -9
[2025-04-21T03:02:42.728-0300] {standard_task_runner.py:190} ERROR - ('Job 49 was killed before it finished (likely due to running out of memory)', 'For more information, see https://airflow.apache.org/docs/apache-airflow/stable/troubleshooting.html#LocalTaskJob-killed')
[2025-04-21T08:56:13.375-0300] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-21T08:56:13.384-0300] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: DW.Duck scheduled__2025-04-20T00:00:00+00:00 [queued]>
[2025-04-21T08:56:13.390-0300] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: DW.Duck scheduled__2025-04-20T00:00:00+00:00 [queued]>
[2025-04-21T08:56:13.390-0300] {taskinstance.py:2866} INFO - Starting attempt 1 of 4
[2025-04-21T08:56:13.406-0300] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): Duck> on 2025-04-20 00:00:00+00:00
[2025-04-21T08:56:13.412-0300] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'DW', 'Duck', 'scheduled__2025-04-20T00:00:00+00:00', '--job-id', '60', '--raw', '--subdir', 'DAGS_FOLDER/DW.py', '--cfg-path', '/tmp/tmplnqie49y']
[2025-04-21T08:56:13.413-0300] {standard_task_runner.py:105} INFO - Job 60: Subtask Duck
[2025-04-21T08:56:13.417-0300] {logging_mixin.py:190} WARNING - /home/lenovo/miniconda3/envs/worker/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=35366) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-04-21T08:56:13.417-0300] {standard_task_runner.py:72} INFO - Started process 35462 to run task
[2025-04-21T08:56:13.443-0300] {task_command.py:467} INFO - Running <TaskInstance: DW.Duck scheduled__2025-04-20T00:00:00+00:00 [running]> on host DESKTOP-1GIT60U.
[2025-04-21T08:56:13.496-0300] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='DW' AIRFLOW_CTX_TASK_ID='Duck' AIRFLOW_CTX_EXECUTION_DATE='2025-04-20T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-04-20T00:00:00+00:00'
[2025-04-21T08:56:13.497-0300] {taskinstance.py:731} INFO - ::endgroup::
[2025-04-21T08:59:05.115-0300] {local_task_job_runner.py:346} WARNING - State of this instance has been externally set to None. Terminating instance.
[2025-04-21T08:59:05.116-0300] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-04-21T08:59:05.118-0300] {process_utils.py:132} INFO - Sending 15 to group 35462. PIDs of all processes in the group: [35462]
[2025-04-21T08:59:05.118-0300] {process_utils.py:87} INFO - Sending the signal 15 to group 35462
[2025-04-21T09:00:07.309-0300] {process_utils.py:150} WARNING - process psutil.Process(pid=35462, name='airflow task runner: DW Duck scheduled__2025-04-20T00:00:00+00:00 60', status='sleeping', started='08:56:13') did not respond to SIGTERM. Trying SIGKILL
[2025-04-21T09:00:07.310-0300] {process_utils.py:87} INFO - Sending the signal 9 to group 35462
[2025-04-21T09:00:07.323-0300] {process_utils.py:80} INFO - Process psutil.Process(pid=35462, name='airflow task runner: DW Duck scheduled__2025-04-20T00:00:00+00:00 60', status='terminated', exitcode=<Negsignal.SIGKILL: -9>, started='08:56:13') (35462) terminated with exit code -9
[2025-04-21T09:00:07.324-0300] {standard_task_runner.py:190} ERROR - ('Job 60 was killed before it finished (likely due to running out of memory)', 'For more information, see https://airflow.apache.org/docs/apache-airflow/stable/troubleshooting.html#LocalTaskJob-killed')
